### PR TITLE
Article: Python BOOL Function

### PR DESCRIPTION
**Pull request by varunu28**

### CODE REVIEW

File: Python-Function-BOOL.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Python bool(x)
+
+`bool()` is a built-in function in Python 3. This function returns a Boolean value, i.e. one of True or False. It takes one argument `x`.
```
Comment (atjonathan) [inline Python-Function-BOOL.md]: ~~one of True or False.~~ True or False.

~~argument `x`.~~ argument, `x`.

File: Python-Function-BOOL.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+`bool()` is a built-in function in Python 3. This function returns a Boolean value, i.e. one of True or False. It takes one argument `x`. 
+
+## Arguments
+
+It takes one argument, `x`. x is converted using the standard [Truth Testing Procedure](https://docs.python.org/3/library/stdtypes.html#truth). 
+
+## Return Value
+
+If x is false or omitted, this returns False; otherwise it returns True.
```
Comment (atjonathan) [inline Python-Function-BOOL.md]: ~~x~~ `x`

File: Python-Function-BOOL.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Python bool(x)
+
+`bool()` is a built-in function in Python 3. This function returns a Boolean value, i.e. one of True or False. It takes one argument `x`. 
+
+## Arguments
+
+It takes one argument, `x`. x is converted using the standard [Truth Testing Procedure](https://docs.python.org/3/library/stdtypes.html#truth).
```
Comment (atjonathan) [inline Python-Function-BOOL.md]: ~~x is converted~~ `x` is converted

**Comment by atjonathan** [comment]
@alayek please take a look...

File: Python-Function-BOOL.md (hunk 2 above)
Comment (alayek) [inline Python-Function-BOOL.md]:
```diff
- returns False; otherwise it returns True
+ returns `False`; otherwise it returns `True`
```

**Comment by alayek** [comment]
# LGTM :+1:  

Once you have fixed points raised in a comment, let's :shipit: :package:

**Comment by Rafase282** [comment]
Please add the file to the Python functions parent article.

**Comment by varunu28** [comment]
@alayek  @atjonathan I have done the changes. Please check.

**Comment by atjonathan** [comment]
@varunu28 Please check @Rafase282 comment: https://github.com/FreeCodeCamp/wiki/pull/1034#issuecomment-221669249

**Comment by varunu28** [comment]
@atjonathan I am not sure what is meant by "Python functions parent article." Can you please help me with this?

**Comment by alayek** [comment]
@varunu28 [this](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Python-Functions) is the main article, which refers to other articles on Python functions.

Please edit this file as well and include yours, in alphabetical order.

**Comment by varunu28** [comment]
@alayek I have done the changes. Please check.

**Comment by alayek** [comment]
# LGTM :+1:  

Let's :shipit: :package: :boom:

**Comment by Rafase282** [comment]
# LGTM

**Pull request merged**